leapp/messaging/answerstore.py

```diff
@@ -183,9 +193,10 @@ def generate(self, dialogs, answer_file_path):
                         answer_entry += '# {key} = {value}\n'.format(key=component.key, value=default or '')
 
                     f.writelines([
-                        '# {}\n'.format(' {}.{} '.format(dialog.scope, component.key).center(77, '=')),
-                        '# {:<20}{}\n'.format('Label:', component.label),
-                        '# {:<20}{}\n'.format('Description:', component.description),
+                        '# {}\n'.format(' {}.{} '.format(dialog.scope,
+                                                         component.key.replace('\n', '')).center(77, '=')),
```

Comment:
These \n replacement should not be here, sorry, this would be solved differently.